bungiesearch/utils.py

```diff
@@ -8,11 +8,12 @@
 from . import Bungiesearch
 
 
-def update_index(model_items, model_name, bulk_size=100, num_docs=-1, start_date=None, end_date=None):
+def update_index(model_items, model_name, action='index', bulk_size=100, num_docs=-1, start_date=None, end_date=None, commit=True):
     '''
     Updates the index for the provided model_items.
     :param model_items: a list of model_items (django Model instances, or proxy instances) which are to be indexed, or updated.
     :param model_name: doctype, which must also be the model name.
+    :param action: the action that you'd like to perform on this group of data. Must be in ('index', 'delete') and defaults to 'index'
     :param bulk_size: bulk size for indexing. Defaults to 100.
     :param num_docs: maximum number of model_items from the provided list to be indexed.
     :param start_date: start date for indexing. Must be as YYYY-MM-DD.
```

Comment:
Could you add a tiny description of what the `commit` parameter does (although it's obvious, it helps when reading the docs)?